packages/api-specification/src/hiddenWindow.html

```diff
@@ -0,0 +1,7 @@
+<!doctype html>
+<body>
+  <script src="window.js"></script>
+  <script>
+    ssf.window('http://localhost:5000/index.html', 'name', 'child=yes');
```

Comment:
As this loads the main application, would something like _mainWindow_ be better?